.github/workflows/go.yml

```
@@ -0,0 +1,34 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    strategy:
+      matrix: 
+        go-version: [1.12.x, 1.13.x, 1.14.x]
```

The variable should be either be `go-versions` as used below in [L22](https://github.com/kubernetes-sigs/yaml/pull/49/files#diff-678682767f2477de3e3c546746f8568b9a1942b2c647d32331d7e774b8ff8d9fR22) or the variable name in that line needs to be changed.

```suggestion
        go-versions: [1.12.x, 1.13.x, 1.14.x]
```

---

i thought i used the plural in both places. Fixed now